src/Adapter/MSTestAdapter.PlatformServices/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.ObjectModel.ITestMethod.TestIdGenerationStrategy.get -> Microsoft.VisualStudio.TestTools.UnitTesting.TestIdGenerationStrategy
```

Comment:
move to shipped?

Reply:
Will keep it as unshipped for now and if I see the same problem of "we forgot to move it", I will revise and use only shipped file. Thanks!